### ISSUE TITLE
tfjs-node backend not recognized while using coco-ssd model on node.js

### ISSUE THREAD

**Issue by qozle** [tensorflow/tfjs#4234]
Hi all.  Been banging my head against this for a while now.  Opened a stackoverflow question [here](https://stackoverflow.com/q/64832338/10129202).  

I keep getting this happy message:
```
============================
Hi there 👋. Looks like you are running TensorFlow.js in Node.js. To speed things up dramatically, 
install our node backend, which binds to TensorFlow C++, by running npm i @tensorflow/tfjs-node, 
or npm i @tensorflow/tfjs-node-gpu if you have CUDA. Then call require('@tensorflow/tfjs-node'); (-
gpu suffix for CUDA) at the start of your program. Visit https://github.com/tensorflow/tfjs-node for 
more details.
============================
```

I'm using the 2.70 tfjs-node npm package and the 2.1 @tensorflow-models/coco-ssd model npm package on an ubuntu 20 LTS server.  Here's how I'm importing it:
```
const tf = require("@tensorflow/tfjs-node");
const cocoSsd = require("@tensorflow-models/coco-ssd");
tf.enableProdMode();
```
Just in case, I've made sure I'm using python 2.7 as my default python version.
```
 $ python -V
Python 2.7.18
```
I tried uninstall both packages and deleting node_modules, and then reinstalling, making sure to install tfjs-node first.  

Here's the code that brings up the issue:
```
const image = await tf.node.decodeImage(resp.body, 3);
const predictions = await nsfwModel.classify(image);
const catObjects = await catModel.detect(image);
image.dispose();
```
where
```
catModel = await cocoSsd.load();
```
I'm also using the nsfwjs module (@nsfwjs)

Also this is my first time submitting an issue to a big project so, much love, hope this isn't a duplicate or something really dumb I'm missing.

**Comment by rthadur** [comment]
@qozle can you please share your package.json file

**Comment by qozle** [comment]
Certainly: 

```
{
  "name": "twit-server",
  "version": "1.0.0",
  "description": "back-end server for the twit portfolio item website",
  "main": "server.js",
  "scripts": {
    "test": "echo \"Error: no test specified\" && exit 1",
    "start": "cross-env NODE_ENV=prod node server.js",
    "start-dev": "cross-env NODE_ENV=dev node server.js"
  },
  "author": "Max flanagan",
  "license": "ISC",
  "dependencies": {
    "@rodrigogs/mysql-events": "^0.6.0",
    "@tensorflow-models/coco-ssd": "^2.1.0",
    "@tensorflow/tfjs-node": "^2.1.0",
    "cors": "^2.8.5",
    "cross-env": "^7.0.2",
    "dotenv": "^8.2.0",
    "express": "^4.17.1",
    "https": "^1.0.0",
    "mysql": "^2.18.1",
    "needle": "^2.5.2",
    "nsfwjs": "^2.2.0",
    "ws": "^7.3.1"
  }
}
```

**Comment by qozle** [comment]
Also, I noticed this (I had it flagged to not show up but after a system restart, it showed up again):

```
2020-11-16 19:03:33.911816: I tensorflow/core/platform/cpu_feature_guard.cc:142] Your CPU supports instructions that this TensorFlow binary was not compiled to use: AVX2 FMA
2020-11-16 19:03:33.959836: I tensorflow/core/platform/profile_utils/cpu_utils.cc:94] CPU Frequency: 2199995000 Hz
2020-11-16 19:03:33.960352: I tensorflow/compiler/xla/service/service.cc:168] XLA service 0x42de320 initialized for platform Host (this does not guarantee that XLA will be used). Devices:
2020-11-16 19:03:33.960532: I tensorflow/compiler/xla/service/service.cc:176]   StreamExecutor device (0): Host, Default Version
2020-11-16 19:03:35.578881: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 8640000 exceeds 10% of system memory.
2020-11-16 19:03:35.599360: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 8640000 exceeds 10% of system memory.
2020-11-16 19:03:35.605278: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 8640000 exceeds 10% of system memory.
2020-11-16 19:03:35.610094: W tensorflow/core/framework/cpu_allocator_impl.cc:81] Allocation of 8640000 exceeds 10% of system memory.
```

**Comment by qozle** [comment]
With this fix now pulled, how can I integrate this updated code into my project?  The npm package version didn't change, so I tried pulling directly from github.  `npm install tensorflow/tfjs-models` worked, but then I couldn't isolate coco-ssd from it.  I tried:

```
const cocoSsd = require("tensorflow-models/coco-ssd")
``` 
and 
```
const cocoSsd = require("tensorflow/tfjs-models/coco-ssd")
```
and 
```
const cocoSsd = require("tensorflow/tensorflow-models/coco-ssd")
```
and
``` 
const cocoSsd = require("tensorflow/coco-ssd")
```
and basically every other variant I could think of.  I even tried `require("tensorflow-models")` as that's the name set in my package.json file now, and even that didn't work.  So I couldn't even require the entire set of models??  It's as if I didn't even install it and yet it's there in my package.json and I definitely just installed it.


Do I just wait until the npm package is updated?

Sorry to post this here, I've tried consulting the npm documentation and can't figure this out.
